### PR TITLE
added apply frontend option for models like jarvis.umdl

### PR DESCRIPTION
**Pull request by hanzala123**

### CODE REVIEW

**Comment by JarbasAl** [comment]
i am not familiar with this option, what is it supposed to do? any docs i can read?

the PR looks good, but i don't know what I need to test

File: jarbas_wake_word_plugin_snowboy/__init__.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ def __init__(self, key_phrase="hey mycroft", config=None, lang="en-us"):
         super().__init__(key_phrase, config, lang)
 
         # load models
+        ApplyFrontend = self.config.get("apply_frontend", False)
```
Comment (JarbasAl) [inline jarbas_wake_word_plugin_snowboy/__init__.py]: snake_case instead of CamelCase for variables please

**Comment by hanzala123** [comment]
https://github.com/seasalt-ai/snowboy/blob/master/include/snowboy-detect.h#L126
here is the documetation.

**Comment by hanzala123** [comment]
@JarbasAI 
https://github.com/seasalt-ai/snowboy/blob/master/include/snowboy-detect.h#L126
here is the documetation.

**Comment by JarbasAl** [comment]
thanks! i somehow had never heard about that before

**Comment by hanzala123** [comment]
@JarbasAl  this option is not very well documented
and i apologize for using camelCase.

File: jarbas_wake_word_plugin_snowboy/__init__.py (hunk 1 above)
Comment (JarbasAl) [reply jarbas_wake_word_plugin_snowboy/__init__.py]: nvm, ill do this when pushing to pip. thank you for all the improvements!

**Comment by JarbasAl** [comment]
no need to apologize at all, its just a nitpick that doesn't affect how things work. 

thank you for all the PRs

**Pull request merged**